DatWeatherDoe/UI/Menu Bar/MenuBuilder.swift

```diff
@@ -52,6 +53,8 @@ final class MenuBuilder {
     
     private func buildWeatherText() -> NSMenuItem { emptyMenuItem() }
     
+    private func buildSunRiseSetText() -> NSMenuItem { emptyMenuItem() }
```

Comment:
Sunrise is normally treated as a single word so we can rename this to `buildSunriseSetText`

Reply:
Possibly a bad naming choice on my end. This is meant to be a merge of sunriseText and sunsetText. Hence the "interesting" spelling. It is not intended to mean "setText" (i.e. set the text). I am open to suggestions for better names if you'd like to avoid the confusion.